upgrade-kubernetes-version.md

```diff
@@ -0,0 +1,157 @@
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          3       2         1             69m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          4       2         0             69m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          4       2         0             69m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          4       2         0             69m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          3       2         1             70m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          2       2         2             70m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          2       2         2             70m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          2       2         2             71m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   3          3       2         0             71m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   3          3       2         0             71m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          3       3         1             71m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          3       3         1             71m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          3       3         1             72m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          3       3         1             72m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          3       3         1             72m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          3       3         1             72m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          4       3         0             72m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          4       3         0             72m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   4          4       3         0             75m   v1.25.2
+docker-cluster-one-6gm59   docker-cluster-one   true          true                   3          3       3         0             75m   v1.25.2
+```
+
+That's a console-full! What we see above is the process of a rolling upgrade:
```

Comment:
@jackfrancis  I wonder if watching the machines would explain it better. As then we see the machines being upgraded one by one

(not a blocker, but let's discuss if we want to do that)